modules/ibi-image-based-install-cluster-guide.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+// * edge_computing/image-based-install/ibi-understanding-image-based-install.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ztp-image-based-install-hub-cluster-guide_{context}"]
+= Cluster guidelines for an image-based installation and deployment
+
+For a successful image-based installation and deployment, see the following guidelines.
+
+== Hub cluster guidelines
+
+* If you are using {rh-rhacm-first}, to avoid including any {rh-rhacm} resources in your seed image, you need to disable all optional {rh-rhacm} add-ons before generating the seed image.
+
+== Seed cluster guidelines
+
+* If your cluster deployment at the edge of the network requires a proxy configuration, you must create a seed image from a seed cluster featuring a proxy configuration. The proxy configurations do not have to match.
+
+* If you set an maximum transmission unit (MTU) in the seed cluster, you must the same MTU value in the static network configuration for the image-based configuration ISO.
```

Comment:
```suggestion
* If you set an maximum transmission unit (MTU) in the seed cluster, you must set the same MTU value in the static network configuration for the image-based configuration ISO.
```

Reply:
Thanks Trey, let me know if any other comments, or if you can give your LGTM